client/snapper.cc

```diff
@@ -452,6 +460,11 @@ list_from_one_config(ProxySnapper* snapper, ListMode list_mode, bool show_used_s
     {
 	default_snapshot = snapshots.getDefault();
 	active_snapshot = snapshots.getActive();
+
+	if (show_used_space)
+	{
+	    show_used_space = snapper->isQuotaSupported();
```

Comment:
AFAIS this will still not work if quota is disabled on btrfs. show_used_space will still be true and the getUsedSpace() calls will fail.

Reply:
Now it checks whether quota is enabled or not. So in case of Btrfs with disabled quota the `getUsedSpace()` won't be called. Filesystem without quota support is considered as disabled quota.